index.html

```diff
@@ -2,8 +2,8 @@
     <body>
         <p>Pay close attention!</p>
 
-        <p><a href="./badcode.html" target="_blank">This link was submitted by a community member</a></p>
+        <p><a href="./badcode.html" target="_blank" rel="noopener noreferrer">This link was submitted by a community member</a></p>
```

Comment:
- To cut this back link, add the attribute`rel="noopener"` on the tag used to create the link from the parent page to the child page. This attribute value cuts the link, but depending on the browser, lets referrer information be present in the request to the child page.
- To also remove the referrer information use this attribute value: `rel="noopener noreferrer"`.